prog/test/postgres_resource.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+require_relative "../../lib/util"
+
+class Prog::Test::PostgresResource < Prog::Test::Base
+  semaphore :destroy
+
+  def self.assemble
+    postgres_service_project = Project.create(name: "Postgres-Service-Project") { _1.id = Config.postgres_service_project_id }
+    postgres_service_project.associate_with_project(postgres_service_project)
```

Comment:
If Jeremy's PR https://github.com/ubicloud/ubicloud/pull/2616 merged first, we will not have `associate_with_project`, jfyi

Reply:
Merged

Reply:
:ack: updated this